targets/targets.json

```diff
@@ -6586,7 +6607,20 @@
                 512
             ]
         ],
-        "bootloader_supported": true
+        "bootloader_supported": true,
+        "supported_c_libs": {
+            "arm": [
+                "std",
+                "small"
+            ],
+            "gcc_arm": [
+                "std",
+                "small"
+            ]
+        },
+        "supported_application_profiles": [
+            "bare-metal"
```

Comment:
Can you not build with the full profile?

Reply:
You should be able to, I'm working on changing this and will confirm once I have tested a full build. 👍🏿